prod-refractr.yml

```diff
@@ -828,3 +828,9 @@ refracts:
   - https://bugzil.la/123456: https://bugzilla.mozilla.org/show_bug.cgi?id=123456
   - https://bugzil.la/form.foo: https://bugzilla.mozilla.org/form.foo
   - https://bugzil.la/glonk: https://bugzilla.mozilla.org/buglist.cgi?quicksearch=glonk
+
+# https://mozilla-hub.atlassian.net/browse/SVCSE-140
+- status.relay.firefox.com/: https://www.mozilla.org/
```

Comment:
```suggestion
- www.mozilla.org/: status.relay.firefox.com
```

Reply:
status.relay.firefox.com => https://www.mozilla.org/ & not the inverse (bc refractr schema is backwards)